GSA-TTS/FAC
# Summary Updates

# Summary Updates

Base branch PR: https://github.com/GSA-TTS/FAC/pull/2419

## Changes:
1. Summary HTML - 
    * New design. Table-based and pulls from the generals table, with download links (only the PDF one points anywhere).
    * Data that doesn't exist (additional UEIs/EINs, zero findings submissions) will show in "disabled" boxes. As in they exist, but just show lighter text with no link.
2. Add `django.contrib.humanize` to the installed apps. This gives access to some useful templatetags, like [intcomma](https://docs.djangoproject.com/en/4.2/ref/contrib/humanize/#intcomma). 
3. Main scss:
    * Add a weight for semibold (600). This isn't set to anything by default. 
    * Override `theme-color-base-lighter` to be... even lighter. Used for a single background. 
4. `getkey.py` - a template tag to allow accessing dictionary members with spaces, as though one used `data.'Notes to Sefa'`. Which doesn't exist in Django, hence the template tag. 
5. Minor cleanup. A return here, a comment type there. 

## How to Test:
1. Switch to this branch and run normally.
2. Ensure there is data in your dissemination tables. If not, submit an audit or have Cypress submit an audit (or three or six).
3. Search for an audit. Visit its summary page. 
4. Verify that information is showing correctly. In particular, eyeball:
    * The PDF download link
    * The funky-fresh separated table
    * The download links that were a pain to line up properly.
5. Mess around with the screen size to see if things break significantly. I doubt we will have many mobile users, but it's nice to be inclusive. 

## Screenshot:
![image](https://github.com/GSA-TTS/FAC/assets/91098850/479017ce-9a50-4b67-aaa6-fe6436db9efb)
![image](https://github.com/GSA-TTS/FAC/assets/91098850/04298220-9dc7-4799-9b30-cb83597b386a)

## PR checklist: submitters


## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
